libs/navigation.js

```diff
@@ -2,10 +2,6 @@ $('#control-nav').click(function () {
     window.location.replace('control.html');
 });
 
-$('#notes-nav').click(function () {
-    window.location.replace('notes.html');
-});
-
```

Comment:
why was this removed?

Reply:
I did NOT remove it :P It auto suggested that.

Reply:
> I did NOT remove it :P It auto suggested that.

haha please add it back then otherwise it wont go to the notes

Reply:
HAHAHAHA. I like how u both within few seconds told so much that I cudnt eat without notification buzzing, hahahahaha